--- configure.old	2023-10-03 16:59:13.282425399 -0400
+++ configure	2023-10-03 17:00:44.280695056 -0400
@@ -3445,12 +3445,12 @@
       ;;
    *linux*|*kfreebsd*-gnu*)
       mach="`uname -m`"
-      if test "$mach" = "x86_64" -o "$mach" = "ia86" -o "$mach" = "alpha" -o "$mach" = "ppc64" -o "$mach" = "s390x"; then
+      if test "$mach" = "x86_64" -o "$mach" = "ia86" -o "$mach" = "alpha" -o "$mach" = "ppc64" -o "$mach" = "ppc64le" -o "$mach" = "s390x" -o "$mach" = "aarch64" -o "$mach" = "e2k" -o "$mach" = "riscv64"; then
          bitflag="64"
          osis64bit=yes
       fi
       ;;
-   *-freebsd*)
+   *-freebsd* | *-openbsd* | *-netbsd*)
       mach="`uname -m`"
       if test "$mach" = "amd64"; then
          bitflag="64"
@@ -3568,8 +3568,9 @@
       CFLAGS="${CFLAGS} -m32"
       LDFLAGS="${LDFLAGS} -m32"
    elif test "$bitflag" = "64"; then
-      CFLAGS="${CFLAGS} -m64"
-      LDFLAGS="${LDFLAGS} -m64"
+      echo "Skipping -m64 on this CPU"
+# WRL CFLAGS="${CFLAGS} -m64"
+# WRL LDFLAGS="${LDFLAGS} -m64"
    else
       bitflag="32"
    fi
