--- configure	2021-06-08 16:08:09.716061275 -0400
+++ regina-rexx-3.9.3/configure	2021-06-08 16:15:15.136639683 -0400
@@ -3477,7 +3477,8 @@
 #
 # Now do platform specific tests
 #
-gcc_64bit="-m64"
+#gcc_64bit="-m64"
+gcc_64bit=""
 gcc_32bit="-m32"
 on_osx="no"
 osis64bit=no
@@ -3516,7 +3517,7 @@
       ;;
    *linux* | *kfreebsd*-gnu*)
       mach="`uname -m`"
-      if test "$mach" = "x86_64" -o "$mach" = "ia86" -o "$mach" = "alpha" -o "$mach" = "ppc64" -o "$mach" = "s390x"; then
+      if test "$mach" = "x86_64" -o "$mach" = "ia86" -o "$mach" = "alpha" -o "$mach" = "ppc64" -o "$mach" = "s390x" -o "$mach" = "aarch64"; then
          bitflag="64"
          osis64bit=yes
       fi
